scripts/run_benchmarks.sh

```diff
@@ -36,6 +36,7 @@ PIP_REQUIREMENTS_FILE=requirements.txt
 # ------------- DROP COLUMNS FRO RESULTS -----------------
 # env inputs
 DRY_RUN=${DRY_RUN:-"false"}
+NO_COMPARE=${NO_COMPARE:-"false"}
```

Comment:
why should this be needed? even so this is double negative logic. should be more straightforward as `NO_COMPARE=${COMPARE:-"true"}`